go/vt/vtctl/workflow/server.go

```diff
@@ -332,42 +334,44 @@ func (s *Server) GetWorkflows(ctx context.Context, req *vtctldatapb.GetWorkflows
 		span.Annotate("workflow", workflow.Name)
 		span.Annotate("tablet_alias", tablet.AliasString())
 
-		id, err := evalengine.ToInt64(row[0])
+		id, err := evalengine.ToInt64(row["id"])
```

Comment:
Just a note that with named values the field names are case sensitive and we're not guaranteeing case with `AS` alias clauses in the `SELECT` right now. Should be fine since these are vitess tables and the columns do seem to be all lower case in our defined schema.